src/pages/Chat.tsx

```
@@ -0,0 +1,308 @@
+      time: '2025-03-20 14:35',
+      message: '나도 이것저것 작업하고 있었지! 요즘 바쁘다 ㅋㅋ',
+    },
+  ]
+
+  return (
+    <Layout>
+      <Container>
+        <FriendListBox>
+          <FriendNumber>45명의 친구</FriendNumber>
+          <FriendList>
+            {data.map(({ id, name, introduce, position, status }) => (
```

ESLint

[error] 'status' is defined but never used.

(@typescript-eslint/no-unused-vars)